requirements.txt

```diff
@@ -1,17 +1,17 @@
-numba
-numpy
-librosa
-tqdm
+numba>=0.54.0
+numpy<1.21.0
```

Comment:
why `numpy<1.21.0`? and all the rest ar >=

Reply:
numba has this requirment

Reply:
numba requiremnt